packages/utils/package.json

```diff
@@ -31,7 +31,7 @@
         "@patternfly/react-table": "^5.0.0",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
-        "react-redux": ">=7.0.0",
+        "react-redux": ">=9.1.0",
```

Comment:
Same as the notifications package. Loosen the restriction rather than restricting it to latest versions.